docs/.vuepress/public/presets/toml/no-empty-icons.toml

```diff
@@ -0,0 +1,150 @@
+[buf]
+format = '(with [$symbol($version )]($style))'
```

Comment:
`$version` doesn't need to be conditional with this preset.
```suggestion
format = '(with [$symbol$version ]($style))'
```

Reply:
Because the whole thing is conditional?